views/apps/create.html

```diff
@@ -1,128 +0,0 @@
-          <rect width="10" height="14" x="6" y="6" rx="1.5"></rect>
-          <path
-            d="M8.064 5.064A1.5 1.5 0 018.5 5h7A1.5 1.5 0 0117 6.5v11a1.5 1.5 0 01-.064.436A1.5 1.5 0 0018 16.5v-11A1.5 1.5 0 0016.5 4h-7a1.5 1.5 0 00-1.436 1.064z"></path>
-        </svg>
-        <span id="copy-text">Copy pairing secret</span>
-      </button>
-
-      <button onclick="document.getElementById('pairing-details').classList.toggle('hidden');return false" class="w-full inline-flex items-center justify-center px-3 py-2 cursor-pointer duration-150 transition bg-white text-purple-700 dark:bg-surface-02dp dark:text-neutral-200 border dark:border-white/10 hover:bg-gray-50 dark:hover:bg-surface-16dp bg-origin-border rounded-md focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-primary mb-4 lg:mb-6">
-        <svg class="mr-2" width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"
-          fill="currentColor">
-          <path d="M12 14a2 2 0 100-4 2 2 0 000 4z"></path>
-          <path fill-rule="evenodd"
-            d="M21 12c0 2.761-4.03 5-9 5s-9-2.239-9-5 4.03-5 9-5 9 2.239 9 5zm-5 0a4 4 0 11-8 0 4 4 0 018 0z"
-            clip-rule="evenodd"></path>
-        </svg>
-        QR code
-      </button>
-    {{end}}
-  </div>
-
-  <div class="flex fixed inset-0 items-center justify-center hidden" id="pairing-details">
-    <div onclick="document.getElementById('pairing-details').classList.toggle('hidden');return false" class="fixed inset-0 bg-gray-900 opacity-50"></div>
-    <div class="bg-white dark:bg-surface-02dp p-4 lg:px-6 rounded shadow relative">
-      <h2 class="mb-4 font-semibold text-lg lg:text-xl font-headline dark:text-white">Scan QR Code in the app to pair</h2>
-      <a href="{{.PairingUri}}" target="_blank" id="pairing-link" class="block border-4 border-purple-600 rounded-lg p-4 lg:p-6">
-        <div id="connect-qrcode"></div>
-      </a>
-      <button onclick="document.getElementById('pairing-details').classList.toggle('hidden');return false"
-      class="w-full inline-flex font-semibold items-center justify-center px-3 py-2 cursor-pointer duration-150 transition bg-white text-gray-700 dark:bg-surface-02dp dark:text-neutral-200 dark:border-neutral-800 hover:bg-gray-50 dark:hover:bg-surface-16dp bg-origin-border shadow rounded-md focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-primary mt-4">
-        Close
-      </button>
-    </div>
-  </div>
-</div>
-
-<script type="text/javascript" src="/public/js/qr-creator.js"></script>
-<script type="text/javascript">
-  // notify the opener if present of the success full connect
-  window.addEventListener("load", (event) => {
-
-    // dispatch a success event which can be listened to by the opener or by the app that embeddes the webview
-    // this gives thoes apps the chance to know the user has enabled the connection
-    const nwcEvent = new CustomEvent("nwc:success", { detail: {} });
```

Comment:
@im-adithya this was not moved over - we need to be really careful not to break existing functionality here